cafebabel/templates/base.html

```diff
@@ -4,6 +4,8 @@
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
 
+{% from "_social-networks-macro.html" import social_networks with context %}
```

Comment:
Do you need the context here?

Reply:
Yes, otherwise the context processor is not passed to the macro, I was a bit surprised too!